src/collections/resources/tutorials/beginner-guide-to-contributing-to-meshery-and-mesheryctl.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: "Beginner's guide to contributing to Meshery and mesheryctl"
+thumbnail: ../../../assets/images/tutorials/intro-meshery-ctl.png
```

Comment:
I think you need to change the thumbnail for this one. You seem to have added two duplicate images for [this](https://61feb804f8236b57b571f06d--layer5ng.netlify.app/resources/service-mesh/beginners-guide-to-contributing-to-meshery-and-mesheryctl) and [Intro to mesheryctl](https://61feb804f8236b57b571f06d--layer5ng.netlify.app/resources/service-mesh/an-introduction-to-mesheryctl)